tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -136,3 +143,88 @@ def _test_delete_cluster(client, cluster, region):
     response = client.list_clusters(region=region)
     cluster_names = [c["cluster_name"] for c in response["items"]]
     assert_that(cluster_names).does_not_contain(cluster_name)
+
+
+def test_official_images(region, api_client):
+    client = image_operations_api.ImageOperationsApi(api_client)
+    response = client.describe_official_images(region=region)
+    assert_that(response["items"]).is_not_empty()
+
+
+@pytest.mark.usefixtures("instance")
+def test_custom_image(request, region, os, pcluster_config_reader, api_client):
+    base_ami = retrieve_latest_ami(region, os)
+
+    config_file = pcluster_config_reader(config_file="image.config.yaml", parent_image=base_ami)
+    with open(config_file) as config_file:
+        config = config_file.read()
+
+    stack_name = generate_stack_name("integ-tests-build-image", request.config.getoption("stackname_suffix"))
+    client = image_operations_api.ImageOperationsApi(api_client)
+
+    with soft_assertions():
```

Comment:
does it make sense to keep all of the tests under the soft_assertions?wondering if in most of the cases it makes sense to move ahead with the test if some of the assertions fail

Reply:
Okay, you are saying that it would make more sense to abort the test as soon as we have an error since the next steps would depend on the state of previous calls... I guess it makes sense, I will remove it also from the cluster APIs tests.

Reply:
Actually, now that I look at other integration test, I see that we don't use this context manager. I will remove it from this patch, and in the subsequent PR for the missing cluster instance and compute fleet APIs I will remove it from the cluster tests.